linode/manager
# M3-1204 fix support time formatting

## Purpose

The timestamp for ticket replies was being mangled beyond all recognition. This uses the standard iso format used throughout the app, and fixes the DateTimeDisplay component so that it shows the correct timezone.

## To Test

1. Reply to a support ticket
2. Go to profile and change your timezone
3. Check to make sure that the output time is correct

## Notes

* Change display name for current user to that user's username, rather than "you"
* Creating a support ticket redirects to the new ticket's detail page (matches behavior of drawer in #3739)
* Add `utc()` to DateTimeDisplay. If I have this right, the API's timestamp was being read by DTD in the default timezone, so wasn't displaying correctly.
* Add the ISO format string to `constants.ts` and import it in DTD
* Changes the conditional for rendering attachments (since `const list = []` is truthy)
